composer.json

```diff
@@ -12,15 +12,25 @@
         "emit"
     ],
     "require-dev": {
-        "phpunit/phpunit": "3.7.*"
+        "phpunit/phpunit": "^5.5",
```

Comment:
Agreed! php7 should be a higher priority.

> I think it's better to remove php 5.5 from .travis.yml, leave composer.json as is.

I agree with this. Please fix your travis.yml!